push-build.sh

```diff
@@ -195,15 +200,15 @@ common::stepheader COPY RELEASE ARTIFACTS
 attempt=0
 while ((attempt<max_attempts)); do
   if $USE_BAZEL; then
-    release::gcs::bazel_push_build $GCS_DEST $LATEST $KUBE_ROOT/_output \
-                                   $RELEASE_BUCKET && break
+    release::gcs::bazel_push_build "$GCS_DEST" "$LATEST" "$KUBE_ROOT"/_output \
+                                   "$RELEASE_BUCKET" && break
   else
-    release::gcs::locally_stage_release_artifacts $GCS_DEST $LATEST \
-                                                  $KUBE_ROOT/_output \
+    release::gcs::locally_stage_release_artifacts "$GCS_DEST" "$LATEST" \
+                                                  "$KUBE_ROOT"/_output \
                                                   $FLAGS_release_kind
     release::gcs::push_release_artifacts \
-     $KUBE_ROOT/_output/gcs-stage/$LATEST \
-     gs://$RELEASE_BUCKET/$GCS_DEST/$LATEST && break
+     "$KUBE_ROOT"/_output/gcs-stage/"$LATEST" \
```

Comment:
prefer quoting the whole string if it's a single argument
```suggestion
     "$KUBE_ROOT/_output/gcs-stage/$LATEST" \
```